spring-batch-docs/asciidoc/scalability.adoc

```diff
@@ -260,30 +263,30 @@ Spring Batch creates step executions for the partitions called
 The `PartitionHandler` is the component that
       knows about the fabric of the remoting or grid environment. It is able
       to send `StepExecution` requests to the remote
-      `Steps`, wrapped in some fabric-specific format, like a DTO. It does not
-      have to know how to split up the input data, or how to aggregate the
-      result of multiple `Step` executions. Generally speaking it probably also
-      doesn't need to know about resilience or failover, since those are
-      features of the fabric in many cases, and anyway Spring Batch always
-      provides restartability independent of the fabric: a failed `Job` can
-      always be restarted and only the failed `Steps` will be
+      `Step` implementations, wrapped in some fabric-specific format, like a DTO. It does not
+      have to know how to split the input data or how to aggregate the
+      result of multiple `Step` executions. Generally speaking, it probably also
+      does not need to know about resilience or failover, since those are
+      features of the fabric in many cases. In any case, Spring Batch always
+      provides restartability independent of the fabric. A failed `Job` can
+      always be restarted and only the failed `Steps` are
       re-executed.
 
 The `PartitionHandler` interface can have
-      specialized implementations for a variety of fabric types: e.g. simple
+      specialized implementations for a variety of fabric types, including simple
       RMI remoting, EJB remoting, custom web service, JMS, Java Spaces, shared
-      memory grids (like Terracotta or Coherence), grid execution fabrics
+      memory grids (like Terracotta or Coherence), and grid execution fabrics
       (like GridGain). Spring Batch does not contain implementations for any
       proprietary grid or remoting fabrics.
 
-Spring Batch does however provide a useful implementation of
-      `PartitionHandler` that executes `Steps` locally in
+Spring Batch does, however, provide a useful implementation of
+      `PartitionHandler` that executes `Step` implementations locally in
```

Comment:
`instances` instead of `implementations`

Reply:
Yes, please.